Src/Add-AzureDtlVM.ps1

```diff
@@ -1,9 +1,9 @@
 <#
 .SYNOPSIS 
-    This script adds the specified number of Azure virtual machines in the specified DevTest Lab.
+    This script adds the number of Azure virtual machines in the DevTest Lab.
```

Comment:
This scripts adds the specified number of Azure virtual machines to a DevTest Lab.